tests/mocha/field_textinput_test.js

```diff
@@ -168,7 +168,7 @@ suite('Text Input Fields', function() {
           };
           field.clickTarget_ = document.createElement('div');
           Blockly.mainWorkspace = workspace;
-          Blockly.WidgetDiv.DIV = document.createElement('div');
+          Blockly.WidgetDiv.setDiv(document.createElement('div'));
```

Comment:
These tests didn't fail before because the mocha tests don't have any cleanup to reset the state of `Blockly.WidgetDiv.DIV` (even though the original set didn't work).

Could you please add that cleanup to `setTestTeardown` or file an issue. It's possible that adding the cleanup may reveal other test failures, so if that's the case then an issue would be better.

Reply:
Done, and also refactored the test to use the public createDom() for setup.